app/src/unitTests/java/com/artemzin/qualitymatters/models/PicassoBitmapClientTest.java

```diff
@@ -0,0 +1,56 @@
+package com.artemzin.qualitymatters.models;
+
+import android.support.annotation.NonNull;
+import android.widget.ImageView;
+
+import com.artemzin.qualitymatters.QualityMattersRobolectricUnitTestRunner;
+import com.squareup.picasso.Picasso;
+import com.squareup.picasso.RequestCreator;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(QualityMattersRobolectricUnitTestRunner.class)
+public class PicassoBitmapClientTest {
+
+    @SuppressWarnings("NullableProblems") // Initialized in @Before.
+    @NonNull
+    private ImageView imageView;
+
+    @SuppressWarnings("NullableProblems") // Initialized in @Before.
+    @NonNull
+    private Picasso picasso;
+
+    private String url = "http://url.com";
+
+    @SuppressWarnings("NullableProblems") // Initialized in @Before.
+    @NonNull
+    private PicassoBitmapClient picassoBitmapClient;
+
+    @Test
+    public void downloadInto_should_load_then_fit_then_centerCrop() throws Exception {
```

Comment:
Should be camelCase function name and do you really need the `throws Exception` ?